selfdrive/car/honda/values.py

```diff
@@ -1444,6 +1494,9 @@ class HondaCarInfo(CarInfo):
   },
 }
 
+# Bosch radar with these firmware IDs limit steering availability to the same vehicle conditions as the factory LKAS (i.e. speed and wiper mode).
+STEER_LIMITING_RADAR = [b'36802-THR-A220\x00\x00', b'36802-TJB-A540\x00\x00']
+
```

Comment:
@csouers do you know if all Odyssey 2021-22 have this restriction? Can we remove the FW checking?

Reply:
Yes. Removed the check.